tools/docker/build-env-tf.sh

```diff
@@ -0,0 +1,42 @@
+# TF - bazel build env variables
+
+export GCC_HOST_COMPILER_PATH=$(which gcc)
+# Optimization is controlled with bazel --config=opt, it will set AVX and SSE flags on linux if possible (without the march=native flag)
+# It is disabled in TF2.4 config, you may pass it with --copt='-march=native', see BZL_CONFIG arg or just uncomment this line
+#export CC_OPT_FLAGS="-march=native -Wno-sign-compare"
+export PYTHON_BIN_PATH=$(which python)
+export PYTHON_LIB_PATH="$($PYTHON_BIN_PATH -c 'import site; print(site.getsitepackages()[0])')"
+export TF_ENABLE_XLA=1
```

Comment:
Great. Indeed XLA is required (see [here](https://github.com/remicres/otbtf/issues/24#issuecomment-658538475))